images/fedora/f32/Containerfile

```diff
@@ -1,10 +1,10 @@
 FROM registry.fedoraproject.org/fedora:32
 
-ENV NAME=fedora-toolbox VERSION=32
+ENV TOOLBOX_NAME=fedora-toolbox TOOLBOX_VERSION=32
 LABEL com.github.containers.toolbox="true" \
-      com.redhat.component="$NAME" \
-      name="$NAME" \
-      version="$VERSION" \
+      com.redhat.component="$TOOLBOX_NAME" \
```

Comment:
This should remain as it was. It does not reference to the name of a container but to the Toolbox/Toolbx utility.

Reply:
Thanks @HarryMichal, would changing it to ARGs completely be any better ?

Reply:
Yes, I think using ARGs will be a much better option here.